lib/common.sh

```diff
@@ -99,8 +99,13 @@ export BAREMETAL_OPERATOR_IMAGE=${BAREMETAL_OPERATOR_IMAGE:-"quay.io/metal3-io/b
 # Config for OpenStack CLI
 export OPENSTACK_CONFIG=$HOME/.config/openstack/clouds.yaml
 
-# v1alpha2 var
+# CAPI version
 export V1ALPHA2_SWITCH=${V1ALPHA2_SWITCH:-"false"}
+if [ "${V1ALPHA2_SWITCH}" == true ]; then
+  export CAPI_VERSION=${CAPI_VERSION:-"v1alpha2"}
+else
+  export CAPI_VERSION=${CAPI_VERSION:-"v1alpha2"}
+fi
```

Comment:
Isn't it setting the same value for CAPI_VERSION no matter if condition met or not?

Reply:
good catch, fixed!